src/transpileCodeblocks/compiler.ts

```diff
@@ -53,14 +53,13 @@ export class Compiler {
   }
 
   public compile(files: VirtualFiles) {
-    // console.log(compilerOptions)
-
     this.compilerHost.setScriptFileNames([]);
     for (let [fileName, { code }] of Object.entries(files)) {
       code = code.replace(/^$/gm, '//__NEWLINE__');
       this.compilerHost.writeFile(fileName, code);
     }
-    this.compilerHost.setScriptFileNames(Object.keys(files));
+    const filenames = Object.keys(files);
+    this.compilerHost.setScriptFileNames(filenames);
```

Comment:
I think there's a bunch of these going on for logging purposes - could you undo them so we have the diff a little bit smaller?